### ISSUE TITLE
Destination keeps changing in unload combine mode

### ISSUE THREAD

**Issue by Lefurjah** [Stephan-S/FS19_AutoDrive#761]
I am trying to harvest my fields but everytime my tractors with the trailers go to drop of a load and return to the field the destination where they are to drop off their load the next time changes.  This causes them to either get caught in an area I never wanted them to be in or to just drive through a stop and return to the field still full.

**Comment by Stephan-S** [comment]
You are probably driving with the 'Distribute to folders' setting enabled in the vehicle settings of AutoDrive.
Just set this to no or update your version. There was a bug in an earlier version where this setting was used globally and not per vehicle, which means the setting you last set on any vehicle was the one used for all vehicles

**Comment by Lefurjah** [comment]
Okay I have updated Autodrive but now whatever map I try to play the game
will not load if I have autodrive as a selected mod.  I have tried new and
saved games, multiple maps, deleted autodrive and re-downloaded (from
ingame modhub, modhub.us, and directly from githup.  Any idea what is
causing this.  The game just gets to the loading screen and sits there
spinning.  I let it run for 20min just to be sure and nothing.

Thanks,
Brett
